cores/esp8266/core_esp8266_main.cpp

```diff
@@ -41,6 +41,7 @@ extern "C" {
 
 struct rst_info resetInfo;
 
+#if 0
 extern "C" {
 extern const uint32_t __attribute__((section(".ver_number"))) core_version = ARDUINO_ESP8266_GIT_VER;
```

Comment:
If you remove this, then eboot will no longer be able to print the core version:
https://github.com/esp8266/Arduino/blob/7999f5c29c653e6e0b8c77ea42334df8df256a23/bootloaders/eboot/eboot.c#L21-L33

Reply:
(which i personally use to check that I'm running correct version; but if you think it should be removed i can find some other way)

Reply:
I personally don't mind, I could not find where it was used in the core.